### PR TITLE
Removes some bad songs from the lobby musics but adds some good ones too

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

Removes Danzig, Jojo, Running in the 90s and some other bad songs

But adds Drunken Spessman and some good ones

#### Changelog

:cl:  
rscadd: Added new songs
rscdel: Removed bad songs
/:cl:

### CODE REVIEW

**Comment by overlord12345** [comment]
Again, dont remove jojo

**Comment by swissloaf** [comment]
> Again, dont remove jojo

it gets extremely old, these new ones will be a even better replacement

**Comment by jcatjmeow** [comment]
What the fuck is wrong with you people

A t o m i z e  y o u r  p r s

Pr1: removes xyz
Pr2: adds xyz

**Comment by monster860** [comment]
> Add/remove songs from this list individually, rather than multiple at once. This makes it easier to judge PRs that change the list, since PRs that change it up heavily are less likely to meet broad support

> **Add/remove songs from this list individually**

> **individually**

> ### **individually**

> ## **individually**

what part of individually do you not understand

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -10,24 +10,22 @@
 		"https://soundcloud.com/jeffimam/title-plasma-attack", 				// Jeff Imam - Title - Plasma Attack
 		"https://www.youtube.com/watch?v=KaOC9danxNo", 						// David Bowie - Space Oddity (Cover by Chris Hadfield)
 		"https://www.youtube.com/watch?v=_lAfMT5FIZE", 						// Clint Mansell - Moon OST #1 - Welcome to Lunar Industries
-		"https://www.youtube.com/watch?v=f2cGxy-ZHIs", 						// Ólafur Arnalds - So Close (feat. Arnór Dan)
```
Comment (monster860) [inline yogstation/code/controllers/subsystem/ticker.dm:13]: fuck no

**Pull request closed without merging**